src/Sulu/Bundle/AdminBundle/Resources/public/js/components/content-tabs/main.js

```diff
@@ -85,6 +85,17 @@ define([], function() {
 
             // load component after click
             this.sandbox.on('husky.tabs.header.item.select', this.startTabComponent.bind(this));
+
+            // load component after click
```

Comment:
Don't really know what the line below this one does, but I am quite sure this comment is wrong :-)